Packs/OktaASA/Integrations/OktaASA/OktaASA.py

```diff
@@ -177,29 +172,28 @@ def is_token_expired(expires_date: str) -> bool:
     """
     current_utc_time = datetime.now(pytz.utc)
     expires_datetime_date = dateparser.parse(expires_date, settings={'TIMEZONE': 'UTC'}) or current_utc_time
+    # Note: True life time of token is actually 60 mins - we take minutes of 57 minutes.
     expires_datetime_date = expires_datetime_date - timedelta(hours=0, minutes=3)
-    expires_datetime: datetime = arg_to_datetime(expires_date) or current_utc_time
-    return current_utc_time > expires_datetime
+    return current_utc_time > expires_datetime_date
 
 
-def add_time_to_events(events: List[Dict] | None):
+def add_time_to_events(events: List[Dict]):
     """
     Adds the _time key to the events.
     Args:
         events: List[Dict] - list of events to add the _time key to.
     Returns:
         list: The events with the _time key.
     """
-    if events:
-        for event in events:
-            create_time = arg_to_datetime(arg=event.get('timestamp'))
-            event['_time'] = create_time.strftime(DATE_FORMAT) if create_time else None
+    for event in events:
+        create_time = arg_to_datetime(arg=event.get('timestamp'))
+        event['_time'] = create_time.strftime(DATE_FORMAT) if create_time else None
 
 
 '''COMMAND FUNCTIONS'''
 
 
-def test_module(client: OktaASAClient, team_name: str) -> str:
+def test_module(client: OktaASAClient, args: dict) -> str:
```

Comment:
Why do we need args here? this is a test module it wont be called from the war room. We can call get_events with hard coded args or empty args by default.